tests/test_efadv.py

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: (GPL-2.0 OR Linux-OpenIB)
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All rights reserved.
+"""
+Test module for efa direct-verbs.
+"""
+
+import argparse
```

Comment:
I see that none of the above imports are in use, except for "import pyverbs.providers.efa.efadv as efa"
(no need for an empty line between it and the test.base import)

Reply:
Done